project-management/src/main/java/life/qbic/projectmanagement/application/authorization/QbicUserDetails.java

```diff
@@ -57,6 +57,17 @@ public String getUsername() {
     return user.id();
   }
 
+  /**
+   * Returns the unique user display name displayed to the user. Has to be distinguished from the
+   * Spring default method getUserName() which returns the userId to which the ACL permissions are
+   * linked
+   *
+   * @return the unique user display name defined to the user
+   */
+  public String getUserDisplayName() {
```

Comment:
scheme is #property()

Reply:
maybe we go for #platformUserName, display is a little bit confusing i have to admin although I have suggested it (shame on me)